Reflections/Week_11_Reflection.md

```diff
@@ -0,0 +1,26 @@
+1. What have you accomplished this week? (Please list each accomplishment, and explain)
+  - Got the old system working on the lonboard. But, this only worked moving forward. The ystem can't suport more than 2.5 lb and the longboard needs more than that to see a difference in its turing capability.
+  - Made the decision to switch from a linear acuator since the part I need is coming from Denmark and I can't rely in it coming in time. I have deciced to move into using a converbelt as the weight shifting part.
+  - created a gear system that shows off the speed difference between using only the motor, using one set of gears, and two sets of gears. With each incrimenting in speed respeciivly. 
+2. What challenges or difficulties did you face? If you solved them, how? If not, what have you learned so far? Have you sought help or other resources?
+  - I faced difficulty figuring out an alterative to the linear actuator since it was the original idea for the weight shifting part. A converbelt is the next thing, which could be difficult since the belt needs to be tight enouch to not buckle under the wieght. 
+3. What do you plan to accomplish in the following week? (Please list at least 3 concrete "S.M.A.R.T." goals, along with estimated number of hours to complete, or day to complete)
+  - Make a prototype conver belt
+    - Specific: Make a prototype that can show it can move a weight manually by hand
+    - Measurable: Use the 2.5 lb weight to see if the belt can supoort it, and it cna be moved by manually turing the axel by hand.
+    - Achievable: This is kind achievable. The hard part is going to be making the belt. The belt needs to be tight enough to support the wiehgt, but not to tight to not be able to spin on the wheel system.
+  - Find conver belt examples
+    - Specific: Try to find lego conver belts on youtube or otherplaces to get ideas on how to create one that's best for my system.
+    - Measurable: 1-3 links should be what I aim for.
+    - Achiveable: This can be done within the next report out. 
+  - See if i can renforce the wheel system.
+    - Specific: The part where the wheels connect to the rest of the system is not stable enough to make sure it can't move side to side. it only needs to be able to move up or down to simulate suspention, similar to car suspension.
+    - Achiveable: This is something i have been doing slowly since i first built the system. With the extra parts it is more achievable.
+    - Measureble: If the wheels become more restriced when it comes to moving the wheels side to side then I know i have done it.
+4. What resources will you use to accomplish your goals for the upcoming week? (please list out the resources)
```

Comment:
Can you be more specific about what type of parts you plan to use? Think of it as if someone wanted to recreate your project what items would they need to buy?

Reply:
What type of gears, wheels...?